lib/Service/ContextService.php

```diff
@@ -252,6 +252,19 @@ public function delete(int $contextId, string $userId): Context {
 		return $context;
 	}
 
+	public function deleteNodeRel(int $nodeId, int $nodeType): void {
+		try {
+			$nodeRelIds = $this->contextNodeRelMapper->getRelIdsForNode($nodeId, $nodeType);
+			foreach ($nodeRelIds as $nodeRelId) {
+				$this->pageContentMapper->deleteByNodeRelId($nodeRelId);
+				$this->contextNodeRelMapper->deleteByNodeRelId($nodeRelId);
+			}
+			
+		} catch (Exception $e) {
+			$this->logger->error('something went wrong while deleting node relation for node id: '.(string)$nodeId. ' and node type '.(string)$nodeType);
```

Comment:
```suggestion
			$this->logger->error('Something went wrong while deleting node relation for node id: ' . (string)$nodeId . ' and node type ' . (string)$nodeType, ['exception' => $e]);
```

Usually it is good to also pass the exception in the log so that the reason and trace can be found. Also I would expect that method to throw the error instead of silencing with just a log message.